engines/cine/main_loop.cpp

```diff
@@ -75,149 +75,136 @@ static void processEvent(Common::Event &event) {
 	case Common::EVENT_WHEELDOWN:
 		g_cine->_keyInputList.push_back(Common::KeyState(Common::KEYCODE_DOWN));
```

Comment:
It would be useful to add `KEYCODE_UP` and `KEYCODE_DOWN` to the keymapper and handle mouse wheel bindings there.